packages/mobile/src/utils/Logger.ts

```diff
@@ -49,12 +49,12 @@ class Logger extends ReactNativeLogger {
       // one time file access permissions which we cannot use here since the library we are using it
       // does not understand that.
       const combinedLogsPath = this.getCombinedLogsFilePath()
-      await RNFS.writeFile(combinedLogsPath, '========React Native Logs========\n')
       if (await RNFS.exists(rnLogsSrc)) {
+        await RNFS.writeFile(combinedLogsPath, '========React Native Logs========\n')
         await RNFS.appendFile(combinedLogsPath, await RNFS.readFile(rnLogsSrc))
       }
-      await RNFS.appendFile(combinedLogsPath, '\n\n========Geth Logs========\n')
       if (await RNFS.exists(gethLogsSrc)) {
+        await RNFS.appendFile(combinedLogsPath, '\n\n========Geth Logs========\n')
         await RNFS.appendFile(combinedLogsPath, await RNFS.readFile(gethLogsSrc))
       }
       return combinedLogsPath
```

Comment:
What happens if `combinedLogsPath` doesn't exist cause neither of these if statements hit?

Reply:
I'm not sure I follow. If neither of the if statements hit, wouldn't the log file just be empty? It looks to me that `combinedLogsPath` should always exist

Reply:
If RNFS.writeFile doesn't get called, the file won't exist.

Reply:
Oh, I see what you mean. I thought the log file already existed somewhere. Reverting my changes.